### ISSUE TITLE
Create hardware recommendations

### ISSUE THREAD

**Issue by mray** [freedombox/freedombox.org#10]
We need to be able to recommend specific hardware solutions that run FreedomBox.
I see there are practical, political and philosophical problems attached to this.

A reasonable compromise would be to have an "optimal" solution be presented (free hardware, ethical supplier, but harder to get, fewer features and more costly) BUT have the most common set of hardware as a default like Raspberry PI from amazon.

* We need to get links that directly lead to shops to buy the hardware. (no idea what our options are to link to bredefined sets of hardware)
* we need description of the hardware plus some context to FBX (what features are not covered by this, what advantages does the hardware have...)
* a conservative price estimation to give a rough idea what amounts are to be expected
* images of the hardware

**Comment by jvalleroy** [comment]
Here are some general links to get started:
* OLinuXino A20
https://www.olimex.com/Products/OLinuXino/A20/open-source-hardware
* Cubieboard / Cubietruck
http://cubieboard.org/buy/
* Beaglebone Black
http://beagleboard.org/black
* APU / ALIX
http://www.pcengines.ch/order.php
* Raspberry Pi 2
https://www.raspberrypi.org/products/raspberry-pi-2-model-b/

One issue with providing direct links, is that there are different distributors preferred for each region. Should we have a separate pages for NA / SA / EU / Asia / etc?

**Comment by phylophyl** [comment]
we can provide links to homepages, it can be a mess with several languages/versions

**Comment by jonassmedegaard** [comment]
Please remove Raspberry Pi 2 from above list: Makes sense to include most common set of hardware - but hardware which is usable at all with Debian.  I believe that means the most common set of hardware is Beaglebone Black.

(contrib or non-free is *not* Debian, only available as convenience add-ons to Debian)

**Comment by petterreinholdtsen** [comment]
I agree with Jonas that hardware which do not work without non-free stuff (which I suspect is his intention, even though he talk about what is in Debian) should not be recommended on the same level as hardware that do work without any non-free stuff.  If a given piece of hardware depend on free software currently missing in Debian, we can fairly easily fix it.  If the hardware is like RPi and depend on non-free binary blobs, it is a lot harder to fix (allthough someone is trying to create replacement firmware as free software for the RPi).

But instead of removing RPi and other hardware platforms where we do not really know what is running in the background (which is the result of loading proprietary firmware blobs, but also by running proprietary virtual machine solutions), I suggest we split the list in two, and have a list of recommended hardware and another list of hardware where freedombox works when some non-free stuff is added to the mix.  This way we make the set of documented freedombox platforms and set of possible users as large as possible while still making it clear that there is a risk involved with using the non-free platforms.

**Comment by jonassmedegaard** [comment]
I am not against documenting that FreedomBox runs on hardware unsupported by Debian - but the topic of this issue is _recommendations_

**Comment by mray** [comment]
Maybe "recommendation" isn't the right word. What we need is a *reference* for documentation we provide. In that regard the most important metric seems to be what most visitors are probably owning.

We can still make beaglebone our recommended hardware set if Raspberry Pi is particularly unfit for that role.

**Comment by SunilMohanAdapa** [comment]
I started to work on identifying components for "Kits" to be possibly distributed by the Foundation.  Please see the attachment which is very relevant to the discussion.
[kit.pdf](https://github.com/freedombox/freedombox.org/files/273879/kit.pdf)

**Comment by mray** [comment]
Nice to see progress there! I suggest adding tasks somewhere more approachable - maybe as tagged github issues. Tasks inside a PDF won't get much attention.

I might add that we also could need images of all components and the whole set as well as everything build together. Ideally fro a consistent angle on a white, well lit background.

**Comment by SunilMohanAdapa** [comment]
I have sent the document over to people at the foundation.

On 05/22/2016 01:01 AM, mray wrote:
> Nice to see progress there! I suggest adding tasks somewhere more
> approachable - maybe as tagged github issues. Tasks inside a PDF won't
> get much attention.

Indeed.  I will create a GitHub issues after some more comments/agreement.

> 
> I might add that we also could need images of all components and the
> whole set as well as everything build together. Ideally fro a consistent
> angle on a white, well lit background.
> 

I have added this to the list of tasks.

-- 
Sunil

**Comment by Pk13055** [comment]
@jvalleroy I have added the relevant links to the mock-up. Should RPi2 be removed as the 'recommended' hardware solution?  Can you direct me to a format for the 'Buy now' page? I'll try to base it off that.

**Comment by mray** [comment]
Having a single link that points to a shop that offers a complete set would be great. For now even amazon would be great I think.

**Comment by jonassmedegaard** [comment]
Quoting mray (2017-03-11 22:08:40)
> Having a single link that points to a shop that offers a complete set 
> would be great. For now even amazon would be great I think.

Sorry - what are you talking about?  Please provide some context.

 - Jonas

-- 
 * Jonas Smedegaard - idealist & Internet-arkitekt
 * Tlf.: +45 40843136  Website: http://dr.jones.dk/

 [x] quote me freely  [ ] ask before reusing  [ ] keep private

**Comment by mray** [comment]
I'm talking about having hardware recommendations to point to.
We need to be able to point to something specific that is easy to obtain and set up.
A non-technical person should be able to easily follow the way we are laying out. Product names of single board computers only make sense to people that know what power supply they need, what casing they can be put in and what kind of SDcard may be needed to host th operating system.

Something like this [amazon Raspberry Pi set](http://ur1.ca/qmmog) is what we need to be able to point to.

**Comment by jonassmedegaard** [comment]
Quoting mray (2017-03-11 23:57:36)
> I'm talking about having hardware recommendations to point to.
> We need to be able to point to something specific that is easy to obtain and set up.
> A non-technical person should be able to easily follow the way we are laying out. Product names of single board computers only make sense to people that know what power supply they need, what casing they can be put in and what kind of SDcard may be needed to host th operating system.
> 
> Something like this [amazon Raspberry Pi set](http://ur1.ca/qmmog) is what we need to be able to point to.

Sounds sensible.

I am involved in a _Debian_ project called Freedombox - could we please 
move the conversation to either the _Debian_ mailinglist or (created as 
needed) a _Debian_ bugreport?

 - Jonas

-- 
 * Jonas Smedegaard - idealist & Internet-arkitekt
 * Tlf.: +45 40843136  Website: http://dr.jones.dk/

 [x] quote me freely  [ ] ask before reusing  [ ] keep private

**Comment by mray** [comment]
> I am involved in a _Debian_ project called Freedombox - could we please move the conversation to either the _Debian_ mailinglist or (created as needed) a _Debian_ bugreport?

Sure, please bring up this topic infront of any audience that might be of help. I'm actually just interested in getting any *single* link to point to that makes buying FreedomBox hardware possible.

**Comment by jonassmedegaard** [comment]
Quoting mray (2017-03-12 11:22:47)
> > I am involved in a _Debian_ project called Freedombox - could we please move the conversation to either the _Debian_ mailinglist or (created as needed) a _Debian_ bugreport?
> 
> Sure, please bring up this topic infront of any audience that might be of help. I'm actually just interested in getting any *single* link to point to that makes buying FreedomBox hardware possible.

I did not raise and am not the driver of this issue.

I will not actively contribute to this issue at Github.

 - Jonas

-- 
 * Jonas Smedegaard - idealist & Internet-arkitekt
 * Tlf.: +45 40843136  Website: http://dr.jones.dk/

 [x] quote me freely  [ ] ask before reusing  [ ] keep private
